packages/flutter_test/test/accessibility_test.dart

```diff
@@ -154,64 +203,67 @@ void main() {
 
     testWidgets('offscreen text is skipped', (WidgetTester tester) async {
       final SemanticsHandle handle = tester.ensureSemantics();
-      await tester.pumpWidget(_boilerplate(
-        Stack(
-          children: <Widget>[
-            Positioned(
-              left: -300.0,
-              child: Container(
-                width: 200.0,
-                height: 200.0,
-                color: Colors.yellow,
-                child: const Text(
-                  'this is a test',
-                  style: TextStyle(fontSize: 14.0, color: Colors.yellowAccent),
-                ),
+      await tester.pumpWidget(_boilerplate(Stack(
+        children: <Widget>[
+          Positioned(
+            left: -300.0,
+            child: Container(
+              width: 200.0,
+              height: 200.0,
+              color: Colors.yellow,
+              child: const Text(
+                'this is a test',
+                style: TextStyle(fontSize: 14.0, color: Colors.yellowAccent),
               ),
             ),
-          ],
-        )
-      ));
+          ),
+        ],
+      )));
 
       final Evaluation result = await textContrastGuideline.evaluate(tester);
       expect(result.passed, true);
       handle.dispose();
     });
 
-    testWidgets('Disabled button is excluded from text contrast guideline', (WidgetTester tester) async {
+    testWidgets('Disabled button is excluded from text contrast guideline',
+        (WidgetTester tester) async {
       // Regression test https://github.com/flutter/flutter/issues/94428
       final SemanticsHandle handle = tester.ensureSemantics();
-      await tester.pumpWidget(
-        _boilerplate(
-          ElevatedButton(
-            onPressed: null,
-            child: Container(
-              width: 200.0,
-              height: 200.0,
-              color: Colors.yellow,
-              child: const Text(
-                'this is a test',
-                style: TextStyle(fontSize: 14.0, color: Colors.yellowAccent),
-              ),
+      await tester.pumpWidget(_boilerplate(
+        ElevatedButton(
+          onPressed: null,
+          child: Container(
+            width: 200.0,
+            height: 200.0,
+            color: Colors.yellow,
+            child: const Text(
+              'this is a test',
+              style: TextStyle(fontSize: 14.0, color: Colors.yellowAccent),
             ),
           ),
-        )
-      );
+        ),
+      ));
```

Comment:
nit: add missing trailing comma

Reply:
Not sure why it is missing or required? Styleguide does not seem to suggest that it is needed?

Reply:
Done